tests/test_jobqueue.py

```diff
@@ -596,8 +641,8 @@ async def callback(_):
         if wait:
             assert not task.done()
             ready_event.set()
-            await asyncio.sleep(0.1)
+            await task  # no CancelledError here (see source code of JobQueue.stop for details)
```

Comment:
similar as for the other tests: The point here is to ensure that the scheduled job is (not) awaited, which is not tested by your changes …

Reply:
I don't understand. The task of shutting down the job queue is being awaited here, and not the job itself

Reply:
We want to test that

* `job_queue.stop(wait=True)` waits for the job to finish before shutting down
* `job_queue.stop(wait=False)` cancels the job and shuts down directly

`await (task = job_queue.stop(wait=wait)` just wait for the `job_queue` to shut down, but is indifferent to the running job